frontend-react/src/hooks/UseOrganizationPublicKeys.ts

```diff
@@ -0,0 +1,41 @@
+import { useCallback } from "react";
+
+import {
+    RSApiKeysResponse,
+    servicesEndpoints,
+} from "../config/endpoints/settings";
+import { useAuthorizedFetch } from "../contexts/AuthorizedFetchContext";
+import { useSessionContext } from "../contexts/SessionContext";
+
+const { publicKeys } = servicesEndpoints;
+
+export type UseOrganizationPublicKeysResult = {
+    orgPublicKeys: RSApiKeysResponse;
+    isLoading: boolean;
+};
+
+export const useOrganizationPublicKeys = () => {
```

Comment:
Can we make this the default export?

```tsx
export default function useOrganizationPublicKeys() {
    ...
}
```

Reply:
Is this the pattern we want to use in the hooks going forward?